.gitignore

```diff
@@ -1 +1,3 @@
-.vagrant
+*.iml
```

Comment:
The IDE-specific ignore lines probably belong in  your global .gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
fixed